Access_Data.ipynb

```diff
@@ -0,0 +1,539 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        grid_label='gn'
</pre><p>Did you try to drop this aswell? I have some tools that can interpolate different grids (but that would only make sense if adding e.g. <code>gr</code>  here gives you more models)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oceanhackweek/ohw21-proj-cmip-ard/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>